src/cases/components/allocate-role/allocate-role.component.html

```diff
@@ -0,0 +1,30 @@
+
+  <a *ngIf="roleAllocation!=='Exclusion'" class="govuk-back-link">
+    Back
+  </a>
+  <h1 class="govuk-heading-l" >
+    <span class="govuk-caption-l">{{caption}}</span>
+    {{title}}
+  </h1>
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+      <div class="govuk-radios">
+        <div class="govuk-radios__item">
+          <input class="govuk-radios__input" id="self_allocation" name="roleAllocation" type="radio" value="self">
+          <label class="govuk-label govuk-radios__label" for="self_allocation">
+            {{selfText}}
+          </label>
+        </div>
+        <div *ngIf="includeOther || roleAllocation!='Exclusion'" class="govuk-radios__item">
+          <input class="govuk-radios__input" id="other_allocation" name="roleAllocation" type="radio" value="other">
+          <label class="govuk-label govuk-radios__label" for="other_allocation">
+            {{otherText}}
+          </label>
+        </div>
+      </div>
+    </fieldset>
+  </div>
+  <div class="govuk-form-group">
+    <button class="govuk-button govuk-!-margin-right-1" type="submit">Continue</button>
+      <p><a href="javascript:void(0)">Cancel</a></p>
+  </div>
```

Comment:
would this best to move to src\work-allocation-2\components, if this is only used in work allocation 2.